.github/workflows/tests.yml

```diff
@@ -30,7 +30,7 @@ jobs:
           args: -v
 
   # Copied from draft-release.yml, should be consistent if updating draft-release.yml.
```

Comment:
nit: please update this comment to "build-artifacts-and-draft-release.yml" because the file name has changed.